### PR TITLE
Added pore size distribution parser (.psd files) 

### PR DESCRIPTION
**Pull request by kjappelbaum**
Added parser for ([.psd files](http://www.zeoplusplus.org/examples.html#psd)) , included test case for parser.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@
 .DS_Store
 build/
 dist/
+.idea/
```
Comment (ltalirz) [inline .gitignore]: a leftover perhaps?

File: aiida_zeopp/parsers/plain.py [hunk 2]
```diff
@@ -215,6 +216,55 @@ def parse(cls, string):
...
+            if header_line in line:
+                break
+        else:
+            raise ValueError('Did not find header line in data')
+        # extract histogram data
+        bins, counts, cumulatives, derivatives = [], [], [], []
+        for line in lines[i + 1:]:
+            b, count, cumulative, derivative = line.split()
+            bins.append(float(b))
+            counts.append(float(count))
```
Comment (ltalirz) [inline aiida_zeopp/parsers/plain.py]: Shouldn't a count be an integer?

File: .gitignore (hunk 1 above)
Comment (kjappelbaum) [reply .gitignore]: it's because I used PyCharm for this, which creates a .idea/ folder

File: aiida_zeopp/parsers/plain.py (hunk 2 above)
Comment (kjappelbaum) [reply aiida_zeopp/parsers/plain.py]: Thanks for noticing this, I'll change it to int.

File: .gitignore (hunk 1 above)
Comment (ltalirz) [reply .gitignore]: oh sorry, for some reason I thought you added the folder.
in the .gitignore it's fine of course (I'm a bit tired...)

**Pull request merged**